deployment/terraform/create.go

```diff
@@ -180,62 +182,74 @@ func (t *Terraform) Create(initData bool) error {
 
 func (t *Terraform) setupAppServers(extAgent *ssh.ExtAgent, uploadBinary bool, binaryPath string) {
```

Comment:
setupServers

Reply:
I also thought about this, to me a job server is still an app server in the essence but this might cause a confusion here. Also, `setupServers` is quite general, shall we use `setupMMServers` or something like that?

Reply:
Let's leave it then. But then for the refactoring part, let's name them to `setupMMServer` and `setupJobServer`.